components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/gateway/v1/impl/RedeployApiApiServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package org.wso2.carbon.apimgt.rest.api.gateway.v1.impl;
```

Comment:
Shall we add license headers?

Reply:
https://github.com/wso2/carbon-apimgt/pull/9546/commits/dfbd644d35add8643bb9aa652caba4e12f511b5b